cypress/e2e/buy-shirt.cy.ts

```diff
@@ -0,0 +1,33 @@
+describe("Buy a t-shirt", () => {
+  it("then the t-shirt should be bought", () => {
+    cy.visit("http://automationpractice.com/");
+    cy.get("#block_top_menu > ul > li:nth-child(3) > a").click();
+    cy.get(
+        "#center_column a.button.ajax_add_to_cart_button.btn.btn-default",
+    ).click();
+    cy.get("[style*='display: block;'] .button-container > a").click();
+    cy.get(".cart_navigation span").click();
+
+    cy.get("#email").type("aperdomobo@gmail.com");
+    cy.get("#passwd").type("WorkshopProtractor");
+
+    // Debes completar la prueba ...
+    // Step 8
+    cy.get("#SubmitLogin").click();
+    // Step 9
+    cy.get("[name='processAddress']").click();
+    // Step 10
+    cy.get("#uniform-cgv").click();
```

Comment:
use the id of the input not the div
![image](https://user-images.githubusercontent.com/36433884/181785571-f0a78ec8-e2f9-4107-819b-97d2e67b21cf.png)